src/private/MDMViewControllerTransitionContext.m

```diff
@@ -29,7 +30,7 @@ @implementation MDMViewControllerTransitionContext {
 @synthesize backViewController = _backViewController;
 @synthesize foreViewController = _foreViewController;
 
-- (nonnull instancetype)initWithTransition:(nonnull id<MDMTransition>)transition
+- (nonnull instancetype)initWithTransition:(nonnull id<MDMTransition, TransitionWithCustomDurationDelegate>)transition
```

Comment:
We can't necessarily guarantee that our transition will always conform to TransitionWithCustomDurationDelegate, so remove it here and add a runtime check down below instead.